openhtf/util/__init__.py

```diff
@@ -89,3 +89,17 @@ def InnerFunction():
     InnerFunction()
     return x.result
   """
+
+
+class classproperty(object):
```

Comment:
openhtf/io/frontend/__init__.py line 197. Madsci's snapshot of my remote
caught a bunch of what would have been my changes, specifically the
implementation of the PubSub stuff on the server side.

On Wed, Jul 20, 2016 at 3:28 PM, Fahrzin Hemmati <notifications@github.com>
wrote:

> In openhtf/util/__init__.py
> <https://github.com/google/openhtf/pull/377#discussion_r71618464>:
>
> > @@ -89,3 +89,17 @@ def InnerFunction():
> >      InnerFunction()
> >      return x.result
> >    """
> > +
> > +
> > +class classproperty(object):
>
> not used?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/google/openhtf/pull/377/files/63060505ad22755333e95915279a55022070d428#r71618464>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHxdlhLaRMILeg-_A-LiyIui5LIM-q3Kks5qXqEdgaJpZM4JRQ1T>
> .
>